### ISSUE TITLE
Error: rustdoc output format is too new (v19, need v18) when parsing /tmp/baseline.json (STILL HAPPENING)

### ISSUE THREAD

**Issue by scouten-adobe** [obi1kenobi/cargo-semver-check#124]
### Steps to reproduce the bug with the above code

Run cargo-semver-check-action against a PR.

It appears the nightly build of Rustdoc has a new output format which is not supported by this crate.

### Actual Behaviour

Excerpted from https://github.com/contentauth/c2pa-rs/runs/8216717251?check_suite_focus=true:

```
+ cargo semver-checks check-release --current /tmp/current.json --baseline /tmp/baseline.json
Error: rustdoc output format is too new (v19, need v18) when parsing /tmp/baseline.json

note: a newer version of cargo-semver-checks is likely available
```

### Expected Behaviour

No such error occurs.

I checked and this installed 0.9.2 of the crate, which is the latest as of this writing.

**Comment by scouten-adobe** [comment]
Reopening as a duplicate of #122 since I can't reopen that bug and the intended fix did not occur.

**Comment by obi1kenobi** [comment]
Resolved by publishing v0.10.0 manually. Will remove the problematic build step in the publish pipeline when I have a sec.

Apologies for the delay, I just got back home to my laptop.
